website/server/controllers/api-v3/groups.js

```diff
@@ -1066,6 +1067,31 @@ api.inviteToGroup = {
     let uuids = req.body.uuids;
     let emails = req.body.emails;
 
+    // If party, check the limit of members
+    if (group.type === 'party') {
+      let memberCount = 0;
+
+      // Counting the members that already joined the party
+      memberCount += group.memberCount;
+      // Count how many invitations currently exist in the party
+      let query = {};
+      query['invitations.party.id'] = group._id;
+      let groupInvites = await User
+        .find(query)
+        .select('_id')
+        .exec();
+      memberCount += groupInvites.length;
+      // Counting the members that are going to be invited by email and uuids
+      if (uuids) {
```

Comment:
can you move this if and the one after (`if (email)`) to after the call to `Group.validateInvitations`? Because at this point the uuids and emails params have not been validated and may not be arrays